src/main/java/seedu/Equipment/EquipmentManager.java

```diff
@@ -0,0 +1,39 @@
+package seedu.Equipment;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class EquipmentManager {
+    private final HashMap<String, Equipment> equipmentList = new HashMap<>();
+
+    public void addEquipment(String itemName, String serialNumber, EquipmentType type, double cost, String purchasedFrom, String purchasedDate) {
+        Equipment equipment = new Equipment(itemName, serialNumber, type, cost, purchasedFrom, purchasedDate);
+        equipmentList.put(serialNumber, equipment);
+    }
+
+    public Equipment checkEquipment(String serialNumber) {
```

Comment:
Does it make more sense for user to search for equipment via name? If so then we would need to change this.